packages/@lwc/integration-karma/test/native-shadow/force-shadow-migrate-mode/index.spec.js

```diff
@@ -0,0 +1,146 @@
+import { createElement, setFeatureFlagForTest } from 'lwc';
+import Native from 'x/native';
+import Synthetic from 'x/synthetic';
+import StyledLight from 'x/styledLight';
+
+function doubleMicrotask() {
+    // wait for applyShadowMigrateMode()
```

Comment:
The double wait is due to the usage of MutationObserver? I ask because the lwc:dom tests work with one microtask: `packages/@lwc/integration-karma/test/dom-manual/moving-node-to-document/index.spec.js`

Reply:
Yes, this is due to `applyShadowMigrateMode`. TBH I didn't look super deep into why it's two microtasks rather than just one.